core/ir/riscv64/instr_create_api.h.in

```diff
@@ -93,7 +92,8 @@
  * \param m   The destination memory opnd.
  * \param r   The source register opnd.
  */
-#define XINST_CREATE_store(dc, m, r) instr_create_1dst_1src(dc, OP_c_nop, m, r)
+#define XINST_CREATE_store(dc, m, r) \
+    ((opnd_get_size(m) == OPSZ_4) ? INSTR_CREATE_sw(dc, m, r) : INSTR_CREATE_sd(dc, m, r))
```

Comment:
Do we need any special opcode for other OPSZ?

Reply:
Same as above.